Intersect.Client/Core/Audio.cs

```diff
@@ -1,6 +1,6 @@
-﻿using System;
+using System;
```

Comment:
BOM?

Reply:
There are several references to `Guid` in this file, among other things. My VS does not like me getting rid of this import?